scrabble.js

```diff
@@ -0,0 +1,34 @@
+class Scrabble {
+
+  constructor(word) {
+    this.word = word.toUpperCase();
+  }
+
+  score() {
+    var totalPoints = 0
+    if (this.word == '' || this.word == 'NULL') {
+      return totalPoints = 0
+    } else {
+      for (let i = 0; i < word.length; i++) {

```

Comment:
Prefix `this.` to `word.length` so you are accessing the instance value of `word` otherwise it is `undefined`.